src/Microsoft.AspNetCore.Mvc.ApiExplorer/ApiResponseType.cs

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.AspNetCore.Mvc.ModelBinding;
+
+namespace Microsoft.AspNetCore.Mvc.ApiExplorer
+{
+    /// <summary>
+    /// Possible type of the response body which is formatted by <see cref="ApiResponseFormats"/>.
+    /// </summary>
+    public class ApiResponseType
+    {
+        public ApiResponseType()
+        {
+            ApiResponseFormats = new List<ApiResponseFormat>();
+        }
+
+        /// <summary>
+        /// Gets or sets the response formats supported by this type.
+        /// </summary>
+        public IList<ApiResponseFormat> ApiResponseFormats { get; set; }
```

Comment:
`public IList<ApiResponseFormat> ApiResponseFormats { get; set; } = new List<ApiResponseFormat>();`

Then you wouldn't have to document the ctor.